docs/developers-guide/substreams-sink-files.md

```diff
@@ -0,0 +1,275 @@
+---
+description: StreamingFast Substreams sink files
+---
+
+# `substreams-sink-files` introduction
+
+### Overview
+
+The `substreams-sink-files` tool provides the ability to pipe data extracted from a blockchain to various types of files-based persistence solutions. A few available options include [BigQuery](https://cloud.google.com/bigquery), [Clickhouse](https://clickhouse.com), custom scripts and other related data storage and processing tools accepting a file format. For example, you could extract all of the ERC20, ERC721, and ERC1155 transfers from the Ethereum blockchain and persist the data to a files-based store.
+
+Substreams modules are created and prepared for the sink tool. After the sink tool begins running, automated ingestion tasks can be setup to have BigQuery, or other files-based storage solution, ingest the data.
+
+As an example, you could use `substreams-sink-files` to sink data in `JSONL` format to a [Google Cloud Storage (GCS)](https://cloud.google.com/storage) bucket and configure a BigQuery Transfer job to run every 15 minutes. The scheduled job ingests the new files found in the GCS bucket where the data, extracted by the Substreams, was written.
+
+### Accompanying code example
+
+[The accompanying Substreams module](#) associated with this documentation is responsible for extracting a handful of data fields from the Block object injected into the Rust-based map module. The sink tool processes the extracted blockchain data line-by-line and outputs the data to the files-based persistence mechanism you've chosen.
+
+The accompanying code example extracts four data points from the Block object and packs them into the `substreams.sink.files.v1` protobuf's data model. The data is passed to the protobuf as a single line of plain text, a `CSV` entry, or a `JSONL` element. StreamingFast is working on support for binary formats such as [Avro](https://avro.apache.org/), [Parquet](https://parquet.apache.org/), and others.
+
+## Outline
+
+1. Download code and tools
+2. Run code and verify output
+3. Code walkthrough
+4. Understanding the sink tool
+5. Run tool and verify output
+6. Conclusion and review
+7. Getting help
+
+## Download code and tools
+
+### Install `substreams-sink-files`
+
+**TODO:** <i>Where is the actual binary file? What is the file type/format/extension (if any)?</i>
+https://github.com/streamingfast/substreams-sink-files/releases
+
+### Verify installation
+
+Checking the version of `substreams-sink-files` will produce a message similar to:
+
+```bash
+substreams-sink-files version dev
+```
+
+### Update system `PATH`
+
+If you want to use the `substreams-sink-tool` from anywhere on your system make sure the computer's `PATH` environment variable has been set correctly.
+
+Add the following lines to the computer's `~/.bashrc` configuration file and then restart the shell session.
+
+```bash
+export GOPATH=$HOME/go
+export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+```
+
```

Comment:
Just one section for all of those. `PATH` instructions should only be mentioned, so keep first line and remove all the rest.

Adjust instructions and command output with using `v0.2.0` release version.

Reply:
Content collapsed, subtitles removed, version updated, PATH instructions simplified. Updates available in next commit/push.